src/components/Header/index.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from'react';
+import {Link } from 'react-router-dom';
+import MainLogo from '../../images/RoboLogo.png';
```

Comment:
Change paths for new convention.
```suggestion
import MainLogo from 'images/RoboLogo.png';
```